trusted-committer-article.md

```diff
@@ -0,0 +1,232 @@
+# InnerSource Learning Path: Trusted Committer
+
+## Introduction
+
+- tbd
+
+## Introducing The Trusted Committer Role
+
+The Trusted Committer (TC) role is one of the key roles in an InnerSource
+community. It is both a challenging and a rewarding role to fulfill and it is
+instrumental for the success of any InnerSource community. 
+
+Generally speaking, the TC role is defined by its responsibilities, rather than
+by its privileges. On a very high level, a TC represents the interests of both
+their InnerSource community and the products the community is building. So as
+a TC, you'll have both tech oriented and community oriented responsibilities
+and we'll explore both of these dimensions in the following sections.
+
+Before we go into the details of what a TC actually does, let's spend some time
+contrasting the TC role to other roles in IS and why we think the name is both
+apt and important.
+
+- Contributor
+    - makes contributions
+    - sometimes part of the community
+    - sometimes sent by another part of the organization to develop a feature
+      for it. This is why contributors are sometimes referred to as Guests. 
+    - responsible for "fitting in"/conforming to the host team and it's 
+      expectations and processes
+    - similar to an Apache Developer or committer
+
+- Trusted Committer
+    - Sometimes referred to as a host.
+    - In this analogy, he is both resonsible for building the house and setting
+      the house rules to make sure his guests are comfortable.
+    - In comparison to the Contributor, a TC has rights to push code closer to
+      production
+- Product Owner
```

Comment:
again consider omitting this and only providing the parts needed as it relates to the TC.

Reply:
Do we want to mention anything about mentoring/coaching as part of the TC role?

Reply:
It's OK to share a _bit_ about Product Owner (as a foil for what a TC role is **not** responsible for), but should reference that the full training is in the Product Owner segments of the learning path.

Reply:
It's important to dispel the myth that inner source projects will need to accept every contribution that comes their way.
The trusted committer focuses on quality of incoming code and the product owner focuses on feature/function fit to the project.

Reply:
> Do we want to mention anything about mentoring/coaching as part of the TC role?

We shoud. I have currently put that in the section on _upleveling community members_. I'm thinking about merging that section with either _Reducing the barriers to entry_ or _Keeping the community healthy_.

Reply:
> It's important to dispel the myth that inner source projects will need to accept every contribution that comes their way.

Will do.